eng/common/TestResources/New-TestResources.ps1

```diff
@@ -286,7 +286,21 @@ foreach ($templateFile in $templateFiles) {
         Log "Persist the following environment variables based on your detected shell ($shell):`n"
     }
 
-    $deploymentOutputs = @{}
+    $serviceDirectoryPrefix = $ServiceDirectory.ToUpperInvariant()
+
+    $context = Get-AzContext;
+
+    # Add default values
+    $deploymentOutputs = @{
+        "$($serviceDirectoryPrefix)_CLIENT_ID" = $TestApplicationId;
+        "$($serviceDirectoryPrefix)_CLIENT_SECRET" = $TestApplicationSecret;
+        "$($serviceDirectoryPrefix)_TENANT_ID" = $context.Tenant.Id;
```

Comment:
We should use the $TenantId but I like defaulting that value from AzContext. I actually want to default a few of these values from AzContext like the BaseName from the account.